src/mon/Paxos.cc

```diff
@@ -503,27 +503,25 @@ void Paxos::handle_last(MonOpRequestRef op)
   ceph_assert(g_conf()->paxos_kill_at != 2);
 
   // is everyone contiguous and up to date?
-  for (auto p = peer_last_committed.begin();
-       p != peer_last_committed.end();
-       ++p) {
-    if (p->second + 1 < first_committed && first_committed > 1) {
+  for (const auto& [peon_rank, peon_last_committed] : peer_last_committed) {
```

Comment:
s/peon/peer/

Reply:
After peon sends MMonPaxos::OP_LAST message to the leader, the leader deals with the message by Paxos::handle_last.
Previously, I also want to "s/peon/peer". However, the auto bind name "peon_last_committed" will be the same as "peer_last_committed" in that way.